pusher-api/api-reference.md

```diff
@@ -0,0 +1,392 @@
+Pusher API reference
+=====================
+Pusher is a service that publishes information to the integrator's service. This information is data related to products, purchases, inventory etc. The purpose of this service is to ensure that integrators do not have to poll for data related to specific events. <br/>
+An integrator can subscribe to specific events in the Pusher service. When these events get triggered, Pusher service will publish information corresponding to the event to the integrator's service.
+
+Examples of events:
+* PurchaseCreated - This gets triggered when a purchase gets created.
+* ProductUpdated - This gets triggered when product information gets updated in the product library.<br/>
+  See the list of all [Supported events](#supported-events).
+
+The Pusher service uses Webhooks. Webhooks are custom callbacks used to send data from one application to another when a specific event gets triggered. <br/>
+The other option that Pusher service supports is [AWS SQS](https://aws.amazon.com/sqs/). However, this is currently available only to selected integrators and can be made available to others as per the use case.
```

Comment:
Don't think we should mention `AWS SQS` here. If it's only for limited integrators (and afaik only one).

Reply:
I agree, lets remove it.

Reply:
Sure, we can remove this, but this was inserted because we found this on the Swagger link as an option.